### ISSUE TITLE
Move ASPNet CI to VSTS (dev.azure.com/dnceng/public)

### ISSUE THREAD

**Issue by chcosta** [dotnet/arcade#676]
VSTS public CI builds
- [ ] public CI builds are building in VSTS (dnceng.visualstudio.com)
  - [ ] Using YAML

**Comment by markwilkie** [comment]
@chcosta, are you already clear on which agent requirements are blocking?

**Comment by natemcmaster** [comment]
Things left:

* JDK: Started work here https://dev.azure.com/dnceng/internal/_git/dotnet-helix-machines/pullrequest/179?_a=overview, but abandoned to focus on 2.2 work. Was planning to pick it up again when we get back to this.
* Machine support for code signing
* For testing only, we also need Chrome (to facilitate https://github.com/aspnet/KestrelHttpServer/issues/2880 - cc @JunTaoLuo )

**Comment by chcosta** [comment]
I wasn't aware of the exact list.

Is there a resource from ASP.NET that is available to help make progress on these requirements.  Specifically - 

- "JDK" - this looks close.  The iteration loop for getting an artifact is a bit tedious, but about a week of iteration here.
- "For testing only..." - What was the plan for aspnet w.r.t.  Helix?  This could be done as well (about a week), but some of this is throw-away work.  It would be nice to see a plan so we can get an idea of what the timeline is and where to spend the investment.

Regarding "Machine support for code signing".  I think we need to prioritize this a bit higher and engineering needs to help do some of the heavy lifting here (that's my opinion and I'm not at all making a comment about division of labor or who's doing what with respect to the other artifacts).  There's enough nuance, secrets, etc... that we want to make sure we're following a sane / sustainable path.

I know we've all talked at a high-level that things need to be done, but I don't really have a feel for the current state of the world, who's working on what, where the priority of individual engineering efforts is.  It might be nice to get in a room (again), and walk through the plan / priorities so that we can help make progress.

**Comment by markwilkie** [comment]
Yes, let's chat for 1/2 hour or so. @chcosta, you might setting something up w/ @natemcmaster, @Chrisboh, you, and I?

**Comment by markwilkie** [comment]
After meeting, it turns out that hosted pools work for ASP to build.  Open question whether or not these machines are allowed to be configured to sign.  If this approach works, things get much more simple.

Also, to unblock @natemcmaster, updating the current build/sign pool w/ the JDK and Chrome might be the right thing to do short term.  (cc/ @ulisesh)

@natemcmaster will be looking at this again in earnest in early November.

**Comment by riarenas** [comment]
I created dotnet/core-eng#4428 to track finishing up the JDK work.

**Comment by natemcmaster** [comment]
Status update: aspnetcore is using Azure Pipelines and self-hosted pools for our PR checks, but we are not yet using Arcade YAML templates. Adopting the templates is blocked on https://github.com/dotnet/arcade/issues/674 because the templates are tightly coupled to using the Arcade SDK.
